modules/troubleshoot/partials/debug-bundle/generate-rpk.adoc

```diff
@@ -0,0 +1,264 @@
+ifdef::env-kubernetes[]
+:link-configure-bundles: troubleshoot:debug-bundle/configure/kubernetes.adoc
+endif::[]
+ifndef::env-kubernetes[]
+:link-configure-bundles: troubleshoot:debug-bundle/configure/linux.adoc
+endif::[]
+
+To generate a debug bundle with rpk, you have two options depending on your needs:
+
+- Use `rpk debug bundle`: Run this command directly on each broker in the cluster. This method requires access to the nodes your brokers are running on.
+
+- Use `rpk debug remote-bundle`: Run this command from a remote machine to collect diagnostics data from all brokers in the cluster. This method is ideal when you want to gather data without logging into each node individually.
+
+== Prerequisites
```

Comment:
Do we keep it in plural for consistency?

We only have one prereq

Reply:
Yeah it's consistency and also avoids us having to go back and make it plural if we add more prereqs in future